server/client_server.go

```diff
@@ -1,19 +1,37 @@
 package main
 
 import (
+	"fmt"
 	"net/http"
+	"strconv"
+	"strings"
 
 	"github.com/pkg/errors"
 
+	"github.com/mattermost/mattermost-plugin-confluence/server/serializer"
 	"github.com/mattermost/mattermost-plugin-confluence/server/service"
+	"github.com/mattermost/mattermost-plugin-confluence/server/util"
 	"github.com/mattermost/mattermost-plugin-confluence/server/util/types"
 )
 
 const (
-	PathCurrentUser = "/rest/api/user/current"
-	PathAdminData   = "/rest/api/audit"
+	PathCurrentUser            = "/rest/api/user/current"
+	PathCommentData            = "/rest/api/content/%s?expand=body.view,container,space,history"
```

Comment:
The query params should not be present here, they should be passed separately as options where the API req is being made